Kozariz.Edelveis.Core/Migrations/20230504075419_AddPasswordToUserModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Kozariz.Edelveis.Core.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddPasswordToUserModel : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Password",
```

Comment:
It would be beneficial to add a comment or documentation explaining the purpose of the "AddPasswordToUserModel" migration and why the "Password" column is being added to the "UsersTable" table.

Reply:
This is an obsolete migrations from my previous doings, it must be deleted